types/index.d.ts

```diff
@@ -69,8 +72,11 @@ export type AwsLambdaRuntime =
   | "python3.9"
   | "python3.10"
   | "python3.11"
+  | "python3.12"
+  | "python3.13"
   | "ruby2.7"
-  | "ruby3.2";
+  | "ruby3.2"
+  | "ruby3.3";
```

Comment:
What about `ruby3.4`? https://aws.amazon.com/fr/about-aws/whats-new/2025/03/aws-lambda-support-ruby-3-4/

Reply:
added in [265e2b9](https://github.com/oss-serverless/serverless/pull/50/commits/265e2b9078f240c4e4e2192235adc514bed759d0)

Reply:
It's incorrect to put it in the types before we actually support it here.